### ISSUE TITLE
Cluster

### ISSUE THREAD

**Issue by Ballantines42** [openlayers/openlayers#6756]
When the current zoom of the map is exceeded when a cluster layer first appears, the cluster is an unnecessary advantage

What to do?

**Comment by ahocevar** [comment]
Can you rephrase? It is not clear to me what you mean.

**Comment by Ballantines42** [comment]
My view chen I add Cluster Layer in Map : 

    view = new ol.View({
        projection: 'EPSG:3857',
        center: [637413.7998883142, 5646948.4742997745],
        zoom: 6,
        minZoom: 2,
        maxZoom: 16
    });

When I exceed the zoom 6, the feature.get('features').length is multiply by 2 While there is no other feature in the layer.
My layer include 200 feature when exceed the zoom 6, My layer include 400 feature, is not normal because in my BDD, I have 200 feature.

**Comment by ahocevar** [comment]
Still not clear. Please provide a codepen.

**Comment by fredj** [comment]
Closing due to lack of activity.
